app/policies/enrollment_exemption_policy.rb

```diff
@@ -0,0 +1,75 @@
+# After https://docs.google.com/presentation/d/1SdugYDEOCqFT4rGLR06t0x3lu1WZIXxMh1YT7HYM6No/edit#slide=id.g1143cbcbed_0_1
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | States       | Pending | In progress | Withdrawn | Rejected | Approved | Deregistered | Expired |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | Pending      | N/A     | Yes         | Yes       | Yes      | Yes      | No           | No      |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | In progress  | ?       | N/A         | Yes       | Yes      | Yes      | No           | No      |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | Withdrawn    | No      | No          | N/A       | No       | No       | No           | No      |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | Rejected     | No      | No          | No        | N/A      | No       | No           | No      |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | Approved     | No      | No          | No        | No       | N/A      | Yes          | Yes     |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | Deregistered | No      | No          | No        | No       | No       | N/A          | No      |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+#  | Expired      | No      | No          | No        | No       | No       | No           | N/A     |
+#  +--------------+---------+-------------+-----------+----------+----------+--------------+---------+
+# Converted to text with http://www.tablesgenerator.com/text_tables (via LibreOffice Calc)
+#
+# Statuses are: building, pending, being_processed, approved, rejected, expired, withdrawn
+class EnrollmentExemptionPolicy < ApplicationPolicy
```

Comment:
Is this a dupe of app/policies/flood_risk_engine/enrollment_exemption_policy.rb ? Why do we need both?

Reply:
I'm hoping it will replace [this one](https://github.com/DEFRA/flood-risk-back-office/blob/rails6/app/policies/flood_risk_engine/enrollment_exemption_policy.rb) when we get through the last of the Reform refactor. Should be able to remove it when the Export pages are done.